news/2017-09-08-scorewatch-august-week-4.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Scorewatch: August Week 4"
+date: 2017-09-08 15:00:00 +0200
+---
+
+[WubWoofWolf](https://osu.ppy.sh/u/39828) has gained ranks! Sound the alarms! He also got his new top play by setting the first HDHR FC on [DragonForce - Symphony of the Night](https://osu.ppy.sh/b/985141?m=0) with 99.17%, gaining 671pp! Watching this makes me feel a little happier inside.
+
+<iframe width="854" height="480" src="https://www.youtube.com/embed/x3N-ENSXvK4" frameborder="0" allowfullscreen></iframe>
+
+[Adamqs](https://osu.ppy.sh/u/613592) became the first player to achieve the prestigious HDHR FC on [DragonForce - Revolution Deathsquad](https://osu.ppy.sh/b/66609?m=0), taking an amazing 98.41% HDHR FC and claiming 526pp! If you got 8 minutes to spare, this is something you should watch!
```

Comment:
double dutching on HDHR FC after 98.41%, you've already said it, like 3 words before, they dont need to hear it again

you got -> you've got